site/content/docs/user/using-wsl2.md

```diff
@@ -84,6 +84,8 @@ nodes:
 1. create service `kubectl create service nodeport nginx --tcp=80:80 --node-port=30000`
 1. access service `curl localhost:30000`
 
+Alternatively, see [Helpful Tips for WSL2](/docs/user/using-wsl2/#helpful-tips-for-wsl2)
```

Comment:
nit: just use #helpful-tips-for-wsl2 (then we don't have to patch this when the path changes